pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/default_audit_policy.go

```diff
@@ -0,0 +1,32 @@
+package openshiftkubeapiserver
+
+const defaultAuditPolicy = `
```

Comment:
We should probably come-up with some reasonable defaults, which log external requests, but none of the system components. IIRC SME were working on something like that. I'll reach out and try to get it.